src/screens/Settings/useSettingsController.ts

```diff
@@ -1,50 +1,26 @@
-import { useCallback, useEffect, useState } from 'react';
-
-import { LanguageManager } from '@managers/LanguageManager';
+import { useCallback, useEffect } from 'react';
+import { useNavigation } from '@react-navigation/native';
 
 import { EventService } from '@services/EventService';
 
-export const useSettingButtonController = () => {
-  const [language, setLanguage] = useState<'ua' | 'en'>();
-  const [isLoading, setLoading] = useState<boolean>(false);
-
-  const onPressUA = useCallback(async () => {
-    setLoading(true);
-    setLanguage('ua');
-    await LanguageManager.setLanguage('ua');
-    setLoading(false);
-    LanguageManager.setLanguage('ua');
-
-    EventService.emit('action:change-language', 'ua');
-  }, []);
-
-  const onPressEN = useCallback(async () => {
-    setLoading(true);
-    setLanguage('en');
-    await LanguageManager.setLanguage('en');
-    setLoading(false);
-    LanguageManager.setLanguage('en');
+export const useSettingsController = () => {
+const navigation = useNavigation();
```

Comment:
an offset(tab) is absent

+add an eslint rule for checking/fixing the issue in future